app/src/main/java/com/jerboa/ui/components/reports/PostReportItem.kt

```diff
@@ -86,42 +87,43 @@ fun PostReportItem(
             modifier = Modifier
                 .clickable { onPostClick(postView) },
         ) {
-            PostHeaderLine(
-                post = postView.post,
-                creator = postView.creator,
-                community = postView.community,
-                creatorBannedFromCommunity = postView.creator_banned_from_community,
-                instantScores = InstantScores(
-                    myVote = postView.my_vote,
-                    score = postView.counts.score,
-                    upvotes = postView.counts.upvotes,
-                    downvotes = postView.counts.downvotes,
-                ),
+            PostCommunityAndCreatorBlock(
+                postView = postView,
                 onCommunityClick = onCommunityClick,
                 onPersonClick = onPersonClick,
                 showCommunityName = true,
                 showAvatar = showAvatar,
                 blurNSFW = blurNSFW,
-                voteDisplayMode = voteDisplayMode,
                 fullBody = false,
             )
         }
 
         //  Title + metadata
         PostBody(
-            post = postView.post,
-            read = postView.read,
+            postView = postView,
```

Comment:
These were all "optimizations"

Reply:
Any field that changes in postView will cause it recompose instead only if the relevant field changed

Reply:
Erps, my bad, I'll go through and fix some of these.

Reply:
Actually I'm not sure if its worth it. A lot of these use like 5+ fields from `PostView`, and if that's the case, it just seems like we should pass in the full thing anyway. 

IE, is there any field on PostView that changes frequently enough to justify separating these out? I don't think there is, as instantScores is the main one, but that's separated.